build.sbt

```diff
@@ -0,0 +1,36 @@
+import java.io.File
+lazy val apiModels = project.in(file("api-models"))
+  .settings(
+    name := "api-models",
+    organization := "com.gu",
+    version := "0.1", 
+    scalaVersion := "2.12.11",
```

Comment:
You've got a couple of mentions of Scala 2.13 in the shell script above, is that important?